### DISCUSSION TITLE
dotlist generation

### DISCUSSION THREAD

**Discussion by dagap** [omry/omegaconf#904] [Q&A]
I have a requirement where I am using the `compose` API in FB Hydra (https://hydra.cc/docs/advanced/compose_api/)

Here, the `overrides` parameter is in a dotlist format. I was wondering if there is an easy way to generate this from a supplied YAML file or a dictionary?

**Comment by Jasha10** [comment]
Hi @dagap, could you please give a specific example? Given a yaml file like the below, what output are you looking for?
```yaml
foo: bar
baz:
  qux: frob
  zoink:
    - zap
```
Are you interested in a list of the dotpath addresses of all the nodes in this config?
